src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -1,21 +1,23 @@
 ---
-title: Get around GitHub
+title: People, issues, and PRs
```

Comment:
For search results, I'd love to get GitHub back into this title

Reply:
honestly i agree. after i saw it updated in the nav bar, i thought: "it was cute in theory... but in practice i do love 'Getting around GitHub.'" perfect opportunity to practice my newly acquired, indent-sensitive yaml editing skills